clj/build.clj

```diff
@@ -4,19 +4,28 @@
 (def lib 'com.rusher/scimacs)
 (def version (format "0.1.%s" (b/git-count-revs nil)))
 (def class-dir "target/classes")
-(def basis (b/create-basis {:project "deps.edn"}))
+(def uber-file "target/uber.jar")
+(def basis (b/create-basis {:project "deps.edn" :aliases [:native :native-22.3.1]}))
```

Comment:
We can make this configurable based on some env var or so, but for now we require people to just use 22.3.1